minecraft-modpack/dev-timeline.json

```diff
@@ -0,0 +1,47 @@
+[
+    {
+        "title": "Version 4",
+        "timeline": {

```

Comment:
I think you need to also add `"warning": false` on it to prevent PHP's undefined property. Similar on other version